career-development/ladder.md

```diff
@@ -17,114 +17,150 @@ Here are some examples of other career ladders:
 1. Heavily Team Oriented
 1. High Employee Retention
 1. High Quality Code Base
-1. Open Communication 
+1. Open Communication
 1. Pair-Programs
 
-## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled 
+## Dev Values : Simple, Test-Driven, Iterative, Communicative, Respectful, Skilled
 
 ### Level 1
-#### Simple 
+#### Impact
+- [ ] Contributes positively to the development of one or multiple apps.
+
+#### Simple
 - [ ] Is working on naming variables and learning to extract methods.
-	
+
 #### Test-driven
 - [ ] Checks that all tests pass before merging into a shared branch.
 - [ ] Can understand where test expectations are not met and fix some with the help of more senior developers.
 - [ ] Can describe the testing pyramid and concepts around what makes a good test.
 - [ ] Writes “good” test descriptions: uses nesting and clear contexts/assertions
-	
+
 #### Iterative
 - [ ] Understands red-green-refactor.
-	
+
 #### Communication
 - [ ] Updates Asana.
 - [ ] Comes prepared for meetings.
 - [ ] Uses time tracking tools and vacation notifications.
-	
+
 #### Respectful teammate
 - [ ] On time for meetings.
 - [ ] Cares about their work.
-	
+
 #### Technical skills
 - [ ] Knows about the three main sections of a Git project(working, staging, repository) as well as the most common commands(commit, pull, push)
 - [ ] Uses a common linter such as rubocop
 - [ ] Can write code in at least one language.
 - [ ] Broad knowledge of main CS + XP concepts.
 
 ### Level 2
-#### Simple 	
+
+#### Impact
+- [ ] Capable of providing on-call support for their area including systems that they are not familiar with.
```

Comment:
I would like to reframe the addition of 'on-call' support, as I'm working away from that model and more toward a highly scoped 30 hour week. Perhaps just 'support' vs 'on-call support' there?